src/main/kotlin/blackjack/BlackJackApplication.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack
+
+fun main() {
+    val playerNames = InputView.inputPlayerNames()
+
+    val deck = Deck()
+    val gamePlayer = Players(playerNames.map { Player(PlayerName(it)) })
```

Comment:
```suggestion
    val gamePlayer = Players.of(playerNames)
```

정적 팩토리 메서드로 playerNames를 통하여 만들어 주는 건 어떨까요?